Tools/CISupport/ews-build/steps.py

```diff
@@ -735,9 +736,18 @@ def useLog(self, *args, **kwargs):
         with mock.patch('buildbot.process.remotecommand.RemoteShellCommand', ScrubbedRemoteCommand):
             return super()._dovccmd(*args, **kwargs)
 
-    def run(self):
+    @defer.inlineCallbacks
+    def _fetch(self, _):
```

Comment:
Can we achieve same thing without overriding this internal method? 
Maybe by passing `repourl` to Git().__init__()

Reply:
Sadly, we cannot.

The whole issue here is that we want `git fetch` to use a different URL than `git clone`. If we pass `repourl` to `Git().init()`, then if we don't have the repository on a machine yet, we'll clone from the wrong place.